src/components/VesyolkinAS/VesyolkinASElement.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import GaussianElimElement from '../PerederiyVA/Gauss';
+import Converter from './converter/converter';
+import App from './React-Sweeper-master/src/App';
+
+const STYLE = {
+  'border-radius': '2px',
```

Comment:
Лучше так, у себя в ветке уже исправил
```suggestion
  borderRadius: '2px',
```